docs/5. Integrations/Creating and modifying assistants in Glific.md

```
@@ -0,0 +1,58 @@
+# Creating and modifying assistants in Glific 
+This doc details how to create new assistants, modify the created assistants and how to use these in your Glific flows. 
+
+##Creating a new assistant
```

markdownlint-cli2 (0.22.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)